### PR TITLE
Added multicolor pens for NT Rep, Magistrate, and IAA

### PR DESCRIPTION
**Pull request by Tastyfish**
It was based on the unused clerical borg pen, but redone to use a base class and be customizable (and be nicer to use). Before anyone says anything, yes I intentionally updated the commented out clerical borg code, in case we ever use it again.
* Use it in-hand and choose the color!
* There's even a little color band to show the color!
* Idea from [this thread](http://nanotrasen.se/forum/viewtopic.php?f=12&t=6824).

preview

### CODE REVIEW

**Comment by Tastyfish** [comment]
Ignore that I incorrectly set the second-to-last IAA pen to be green instead of blue in the screenshot. :sweat:

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -314,7 +314,7 @@
 	New()
 		src.modules += new /obj/item/device/flashlight(src)
 		src.modules += new /obj/item/device/flash/cyborg(src)
-		src.modules += new /obj/item/weapon/pen/robopen(src)
+		src.modules += new /obj/item/weapon/pen/multi/robopen(src)
```
Comment (tigercat2000) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: the clerical bot isn't even accessible, kek

Comment (Tastyfish) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: I already mentioned that I was aware of that in the OP, it was just for the sake of addressing every grep result :p

**Comment by Fox-McCloud** [comment]
NIce.

**Comment by FalseIncarnate** [comment]
Choices from a list rather than cycling. I think that might be TOO efficient for bureaucrats.

:+1:

**Comment by Deanthelis** [comment]
GLORY TO NANOTRASEN.
Paperwork Peon +1.

**Comment by NTSAM** [comment]
yeee
:+1:

**Comment by Ar3nn** [comment]
Oh god. Walter will eat this up.
+1 though, I do like the idea.

**Comment by MindLeaker** [comment]
:+1: 
Death by Paperwork honobru death.

**Comment by SigholtStarsong** [comment]
Please yes.

**Comment by Spacemanspark** [comment]
:+1:

File: code/modules/mob/living/silicon/robot/robot_modules.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: Clerical bot; still has the most retarded emag item ever.

Comment (Tastyfish) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: I like that getting emagged was the only way for it to be able to stamp things at all, and then it was given the least useful stamp.

**Pull request merged**